decidim-core/app/controllers/decidim/user_activities_controller.rb

```diff
@@ -22,7 +22,7 @@ def index
     def user
       return unless params[:nickname]
 
-      @user ||= current_organization.users.find_by("LOWER(nickname) = ?", params[:nickname].downcase)
+      @user ||= current_organization.users.find_by("nickname = ?", params[:nickname].downcase)
```

Comment:
Does it makes sense to keep downcasing the nickname if it'll be always on lowecase? I understand that the performance win is like a nanosecond, but the main argument would be to clean up the code (and no keep something just-in-case/cargo culting)
```suggestion
      @user ||= current_organization.users.find_by("nickname = ?", params[:nickname])
```

Reply:
Since this is coming from params, I would keep it with `.downcase`, since we cannot control what user will add.